From f950924e8b460036e98caa4401ab19ce0254ee8e Mon Sep 17 00:00:00 2001
From: Shane Loretz <sloretz@openrobotics.org>
Date: Mon, 21 Mar 2022 14:15:49 -0700
Subject: [PATCH] Set APPEND_PROJECT_NAME_TO_INCLUDEDIR to ON

---
 debian/rules.em | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/debian/rules.em b/debian/rules.em
index 284a071..d566c73 100755
--- a/debian/rules.em
+++ b/debian/rules.em
@@ -37,7 +37,8 @@ override_dh_auto_configure:
 	dh_auto_configure -- \
 		-DCMAKE_INSTALL_PREFIX="@(InstallationPrefix)" \
 		-DCMAKE_PREFIX_PATH="@(InstallationPrefix)" \
-		$(BUILD_TESTING_ARG)
+		$(BUILD_TESTING_ARG) \
+		-DAPPEND_PROJECT_NAME_TO_INCLUDEDIR=ON
 
 override_dh_auto_build:
 	# In case we're installing to a non-standard location, look for a setup.sh
-- 
2.35.3

